src/shared/components/BaseToastMessages.tsx

```diff
@@ -128,25 +90,22 @@ export default function BaseToastMessages({
       >
         {messages.map(message => (
           <li
-            className={classnames(
-              'relative w-full container hover:cursor-pointer',
-              {
-                // Add a bottom margin to visible messages only. Typically, we'd
-                // use a `space-y-2` class on the parent to space children.
-                // Doing that here could cause an undesired top margin on
-                // the first visible message in a list that contains (only)
-                // visually-hidden messages before it.
-                // See https://tailwindcss.com/docs/space#limitations
-                'mb-2': !message.visuallyHidden,
-                // Slide in from right in narrow viewports; fade in larger
-                // viewports to toast message isn't flying too far
-                'motion-safe:animate-slide-in-from-right lg:animate-fade-in':
-                  !message.isDismissed,
-                // Only ever fade in if motion-reduction is preferred
-                'motion-reduce:animate-fade-in': !message.isDismissed,
-                'animate-fade-out': message.isDismissed,
```

Comment:
These transitions are not currently working, but that's not a thing introduced by these changes. I'll continue to investigate.